spdx-utils/src/main/kotlin/SpdxDeclaredLicenseMapping.kt

```diff
@@ -82,7 +89,15 @@ object SpdxDeclaredLicenseMapping {
         """Apache-2.0 */ &#39; &quot; &#x3D;end --
 
  """.trimIndent() to APACHE_2_0.toExpression(),
+        "academic free license (afl)" to AFL_3_0.toExpression(),
+        "alv2" to APACHE_2_0.toExpression(),
```

Comment:
Single-string mappings like this should go to `SpdxLicenseAliasMapping` instead.

Reply:
Removed all single-string mapping out of `SpdxDeclaredLicenseMapping.kt ` into `SpdxLicenseAliasMapping`